addons/website/static/src/snippets/s_share/000.js

```diff
@@ -20,6 +20,11 @@ const ShareWidget = publicWidget.Widget.extend({
                 return href.replace(urlRegex, function (match, a, b, c) {
                     return a + url + c;
                 }).replace(titleRegex, function (match, a, b, c) {
+                    if (href.indexOf('https://wa.me/?text') !== -1) {
```

Comment:
This is not a very robust test.
I would prefer something like: `if ($a.hasClass('s_share_whatsapp'))`.